shell/src/test/java/tachyon/shell/TFsShellUtilsTest.java

```diff
@@ -42,4 +69,175 @@ public void getFilePathTest() throws IOException {
       Assert.assertEquals(expected, result);
     }
   }
+  
+  private void assertFilePath(File file, String path) {
+    Assert.assertEquals(file.getAbsolutePath().compareTo(path), 0);
+  }
+  
+  public static Comparator<File> createFilePathComparator() { 
+    return  new Comparator<File>() {
+      public int compare(File file1, File file2) {
+        // ascending order
+        return file1.getAbsoluteFile().compareTo(file2.getAbsoluteFile());
+      }
+    };
+  }
+  
+  public static Comparator<TachyonURI> createTachyonURIComparator() {
+    return  new Comparator<TachyonURI>() {
+      public int compare(TachyonURI tUri1, TachyonURI tUri2) {
+        // ascending order
+        return tUri1.getPath().compareTo(tUri2.getPath());
+      }
+    };
+  }
+  
+  public int[] resetTachyonFileHierarchy() throws IOException {
+    /**
+     * Generate such local structure
+     *  /testWildCards
+     *  ├── foo
+     *  |    ├── foobar1
+     *  |    └── foobar2
+     *  ├── bar
+     *  |    └── foobar3
+     *  └── foobar4
+     */
+    int[] fileIds = new int[4];
+    mTfs.delete(new TachyonURI("/testWildCards"), true);
+    mTfs.mkdir(new TachyonURI("/testWildCards"));
+    mTfs.mkdir(new TachyonURI("/testWildCards/foo"));
+    mTfs.mkdir(new TachyonURI("/testWildCards/bar"));
+    fileIds[0] = 
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foo/foobar1", 
+            WriteType.MUST_CACHE, 10);
+    fileIds[1] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foo/foobar2", 
+            WriteType.MUST_CACHE, 20);
+    fileIds[2] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/bar/foobar3", 
+            WriteType.MUST_CACHE, 30);
+    fileIds[3] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foobar4", 
+            WriteType.MUST_CACHE, 40);
+    return fileIds;
+  }
+  
+  @Test
+  public void getTachyonURIsTest() throws IOException {
```

Comment:
The logic of this test is the same as `getFilesTest()`, what about having utility methods like 
```
enum FsType {
    TFS,
    LOCAL
}; 

List<String> getPaths(String path, FsType fsType) {
     List<String> ret;
     if (fsType == FsType.TFS) {
         List<TachyonURI> tPaths = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(path));
         ret = new ArrayList<String>(tPaths.size());
         for (TachyonURI tPath : tPaths) {
             ret.add(tPath.getPath());
         }
    } else {
      .....
   }
}

void sortPaths(List<String> paths) {
    ....
}

void resetFsHierarchy(FsType fsType) {
   ....
}
```

and then we can just have one `getPathTest` like 
```
void getPathTest() {
    for (FsType fsType : FsType.values()) {
       resetFsHierarchy(fsType);
       List<String> paths = getPaths(path, fsType);
       sortPaths(paths);
       .....
   }
}
```